test/saithrift/tests/saiacl.py

```diff
@@ -1634,3 +1634,273 @@ def runTest(self):
+           12. Bind the ACL table with VLAN with attribute SAI_VLAN_ATTR_INGRESS_ACL
+           13. Send the IP packet again through port 2 and ensure the packet is dropped.
+        Part2:
+           14. Change the bind point from VLAN to the port 2
+           15. Send the IP packet again through port 2 and ensure the packet is dropped.
+           16. Send the IP packet again through port 3 and ensure the packet is received on port 1 (LAG)
+           17. Remove the ACL attribute and send the packet from port 2 and from port 3 (VLAN) and verify the packet is received on
+               port 1 (LAG)
+        Part3:
+           18. Remove the ACL table and entry.
+           19. Remove the route and the associated router interface, vlan members and the corresponding VLAN from the database.
+        """
+        switch_init(self.client)
+        port1 = port_list[0]
+        port2 = port_list[1]
+        port3 = port_list[2]
+        v4_enabled = 1
+        v6_enabled = 0
+        vlan_id = 100
+        mac_action = SAI_PACKET_ACTION_FORWARD
+        mac1 = ''
+
+        # Creating VLAN ID
+        vlan_oid = sai_thrift_create_vlan(self.client, vlan_id)
+        vlan_member1 = sai_thrift_create_vlan_member(self.client, vlan_oid, port2, SAI_VLAN_TAGGING_MODE_UNTAGGED)
+        vlan_member2 = sai_thrift_create_vlan_member(self.client, vlan_oid, port3, SAI_VLAN_TAGGING_MODE_UNTAGGED)
+
+        attr_value = sai_thrift_attribute_value_t(u16=vlan_id)
+        attr = sai_thrift_attribute_t(id=SAI_PORT_ATTR_PORT_VLAN_ID, value=attr_value)
+        self.client.sai_thrift_set_port_attribute(port2, attr)
+        self.client.sai_thrift_set_port_attribute(port3, attr)
+
+        # Creating Virtual router
+        vr_id = sai_thrift_create_virtual_router(self.client, v4_enabled, v6_enabled)
+
+        lag_id1 = self.client.sai_thrift_create_lag([])
+        sai_thrift_vlan_remove_ports(self.client, switch.default_vlan.oid, [port1])
+        lag_member_id1 = sai_thrift_create_lag_member(self.client, lag_id1, port1)
+
+        # Creating RIF
+        rif_id1 = sai_thrift_create_router_interface(self.client, vr_id, SAI_ROUTER_INTERFACE_TYPE_PORT, lag_id1, 0, v4_enabled, v6_enabled, mac1)
+        rif_id2 = sai_thrift_create_router_interface(self.client, vr_id, SAI_ROUTER_INTERFACE_TYPE_VLAN, 0, vlan_oid, v4_enabled, v6_enabled, mac1)
+
+        addr_family = SAI_IP_ADDR_FAMILY_IPV4
+        ip_addr1 = '10.10.10.1'
+        ip_mask1 = '255.255.255.0'
+        dmac1 = '00:11:22:33:44:55'
+        ip_addr2 = '192.168.0.1'
+        ip_addr3_subnet = '12.12.12.0'
+        dmac2 = '00:11:22:33:44:56'
+
+        # Creating Neighbor, NHop and route
+        sai_thrift_create_neighbor(self.client, addr_family, rif_id1, ip_addr1, dmac1)
+        nhop1 = sai_thrift_create_nhop(self.client, addr_family, ip_addr1, rif_id1)
+        sai_thrift_create_route(self.client, vr_id, addr_family, ip_addr3_subnet, ip_mask1, nhop1)
+
```

Comment:
Test is passing after adding local route 10.10.0.0/16 to rif_id1

Reply:
Addressed